tools/mirror/src/cli.rs

```diff
@@ -15,43 +15,37 @@ enum SubCommand {
     Run(RunCmd),
 }
 
-/// initialize a target chain with genesis records from the source chain, and
-/// then try to mirror transactions from the source chain to the target chain.
 #[derive(clap::Parser)]
 struct RunCmd {
-    /// source chain home dir
     #[clap(long)]
     source_home: PathBuf,
-    /// target chain home dir
+
     #[clap(long)]
     target_home: PathBuf,
-    /// file containing an optional secret as generated by the
-    /// `prepare` command. Must be provided unless --no-secret is given
+
     #[clap(long)]
     secret_file: Option<PathBuf>,
-    /// Equivalent to passing --secret-file <FILE> where <FILE> is a
-    /// config that indicates no secret should be used. If this is
-    /// given, and --secret-file is also given and points to a config
-    /// that does contain a secret, the mirror will refuse to start
+
     #[clap(long)]
     no_secret: bool,
-    /// Start a NEAR node for the source chain, instead of only using
-    /// whatever's currently stored in --source-home
+
     #[clap(long)]
     online_source: bool,
-    /// If provided, we will stop after sending transactions coming from
-    /// this height in the source chain
+
     #[clap(long)]
     stop_height: Option<BlockHeight>,
+
     #[clap(long)]
     config_path: Option<PathBuf>,
 }
 
 impl RunCmd {
-    fn run(self) -> anyhow::Result<()> {
+    //returning a Result instead of unwrapping, to handle any error properly
```

Comment:
Please follow the [rustdoc documentation guidelines](https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html#how-to-write-documentation). Some things that stand out immediately: 

* `=====` is very seldom used -- this is my first time seeing it used for Rust documentation;
* `///` denotes a doc-comment and should be used here; 
* I don't think code comments need to motivate a specific return value as you did here, nor they should be used to keep a changelog to the functions – we have git for that after all.